app/templates/index.html

```diff
@@ -49,99 +49,29 @@ <h5 class="caption-title">Carousel Title Text 3</h5>
-                                    <img src="https://i.imgur.com/mv10HeP.png">
-                                    <br>
-                                    <p class="news">NEWS</p>
-                                </div>
-                                <div class="col-sm-10 entry-info">
-                                    <h4 class="entrytitle">IT Department Weekly Status Meeting</h4>
-                                    <p class="datenlocation">October 18, 2017</p>
-                                    <p class="datenlocation">Posted in Meeting Notes</p>
-                                    <div class="entrytext">Discussion: This is some text. This is some text. This is some text. This is some text.</div>
-                                </div>
-                            </div>
-                        </a>
-                        <a href="" class="scrolllink">
-                            <div class="entry">
-                                <div class="col-sm-2 entry-pic">
-                                    <img src="https://i.imgur.com/bd6VJr4.png">
-                                    <br>
-                                    <p class="events">EVENTS</p>
-                                </div>
-                                <div class="col-sm-10 entry-info">
-                                    <h4 class="entrytitle">IT Department Weekly Status Meeting</h4>
-                                    <p class="datenlocation">October 18, 2017</p>
-                                    <p class="datenlocation">Posted in Meeting Notes</p>
-                                    <div class="entrytext">Discussion: This is some text. This is some text. This is some text. This is some text.</div>
-                                </div>
-                            </div>
-                        </a>
+                        {% for post in posts %}
+                            {% if post.post_type == "meeting_notes" and post.deleted == False %}
+                                <a href="{{ url_for('main.view_post', post_id = post.id) }}" class="scrolllink">
+                                    <div class="entry">
+                                        <div class="col-sm-2 entry-pic">
+                                            <img src="{{ url_for('static', filename='img/meeting_notes_icon.png') }}">
+                                            <br>
+                                            <p class="meetings">MEETINGS</p>
+                                        </div>
+                                        <div class="col-sm-10 entry-info">
+                                            <h4 class="entrytitle">{{ post.title }}</h4>
+                                            <p class="datenlocation">{{ post.date_created.strftime('%B %d, %Y') }}</p>
+                                            <p class="datenlocation">Posted in Meeting Notes</p>
+                                            <div class="entrytext">{{ post.content | safe }}
```

Comment:
To make it look cleaner, we should setup a post.summary and choose the first `X` characters (200?). It's a lot of scrolling right now.

Reply:
Doesn't have to be MVP then, if it will look weird.